### DISCUSSION TITLE
add release note section to the doc

### DISCUSSION THREAD

**Discussion by ariaieboy** [livewire/livewire#4042] [3. Feature ideas]
livewire have a great document but it dont have a release note page so we can see what changed in each version.
yes we have the github release section but for example in the last release of livewire version 2.7.0 we have some changes that there is no document for them yet and if its have documentation we must search for each of them in document to see what these changed really are.
but if we have a release note section we can provide some more details about each change or at least link to the documentation about that change so users can learn and find out what changed in each version

**Comment by joshhanley** [comment]
@ariaieboy some of the new features haven't been documented yet and will be soon, but it was decided that it was better to tag the release so everyone can start using it, rather than be held up waiting for the documentation.

Personally, I don't think having release notes in the docs is a good idea as it means there will then be two places that need to be maintained and kept in sync. Where as at the moment GitHub releases is the source of truth.

Also if you look at Laravel, they only maintain release notes on GitHub, not in the docs too.

One solution could be to bring the docs repo into the main Livewire repo (like Alpine has now), so that way the documentation gets updated in the same PR, which then makes it easier to find the documentation for a new feature.

Hope this helps!

**Reply by ariaieboy** [reply]
yes thats a good apporoach or at least we can provide more info in the PR descriptions

**Reply by joshhanley** [reply]
Yep agreed!
